Decred Wallet/Features/Transactions/TransactionsViewController.swift

```diff
@@ -0,0 +1,246 @@
+//
+//  TransactionsViewController.swift
+//  Decred Wallet
+//
+// Copyright (c) 2018-2020 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+import UIKit
+import Dcrlibwallet
+
+class TransactionsViewController: UIViewController {
+    @IBOutlet weak var headerTopConstraint: NSLayoutConstraint!
+    @IBOutlet weak var headerStackView: UIStackView!
+    @IBOutlet weak var headerBottomConstraint: NSLayoutConstraint!
+    @IBOutlet var txTableView: UITableView!
+    @IBOutlet var txFilterDropDown: DropMenuButton!
+    @IBOutlet var txSortOrderDropDown: DropMenuButton!
+
+    var noTxsLabel: UILabel {
+        let noTxsLabel = UILabel(frame: self.txTableView.frame)
+        noTxsLabel.text = LocalizedStrings.noTransactions
+        noTxsLabel.font = UIFont(name: "Source Sans Pro", size: 16)
+        noTxsLabel.textColor = UIColor.appColors.lightBluishGray
+        noTxsLabel.textAlignment = .center
+        return noTxsLabel
+    }
+
+    var refreshControl: UIRefreshControl!
+    var allTransactions = [Transaction]()
+    var txFilters = [Int32]()
+    let txSorters: [Bool] = [true, false]
+    var filteredTransactions = [Transaction]()
+    var maximumHeaderTopConstraint: CGFloat?
```

Comment:
Move this to before `headerTopConstraint` and rename to `initialPageHeaderTopConstraintValue`.